resources/views/pages/settings/password.twig

```diff
@@ -0,0 +1,36 @@
+{% extends 'pages/settings/settings.twig' %}
+{% import 'macros/form.twig' as f %}
+{% import 'macros/base.twig' as m %}
+
+{% block title %}{{ __('settings.password') }}{% endblock %}
+
+{% block row_content %}
+        <form action="" enctype="multipart/form-data" method="post">
+            {{ csrf() }}
+
+            <div class="row">
+                <div class="col-md-12">
+                    {{ f.info(__('Here you can change your password.')) }}
```

Comment:
These messages should be replaced by for example `settings.change-password`, `settings.old-password` etc